v1/contracts/Voting.sol

```diff
@@ -1,20 +1,80 @@
 pragma solidity ^0.5.0;
 
+pragma experimental ABIEncoderV2;
+
+import "../../common/contracts/Testable.sol";
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
 
 /**
  * @title Voting system for Oracle.
  * @dev Handles receiving and resolving price requests via a commit-reveal voting scheme.
  */
-contract Voting {
+contract Voting is Testable {
+
+    using SafeMath for uint;
+
+    // The current voting round for the contract. Note: this assumes voting rounds do not overlap.
```

Comment:
We should verify with @madrasi that voting rounds don't overlap.

Reply:
I think we're g2g here!